Source/Tools/LinuxEmulation/LinuxSyscalls/GdbServer.cpp

```diff
@@ -1451,6 +1449,13 @@ void GdbServer::StartThread() {
   FEXCore::Threads::SetSignalMask(OldMask);
 }
 
+void GdbServer::CreateThreadCallback(FEX::HLE::ThreadStateObject* ThreadObject) {
+  if (SyscallHandler->TM.GetThreadCount() == 1) {
+    // Sleep the first thread created. This is because FEX only currently supports attaching at startup.
```

Comment:
Wrong word order. I think this is what you mean?

```suggestion
    // Sleep the first thread created. This is because FEX only supports attaching at startup currently.
```